alts/src/main/java/io/grpc/alts/internal/AltsHandshakerClient.java

```diff
@@ -35,6 +35,9 @@
   private static final String APPLICATION_PROTOCOL = "grpc";
   private static final String RECORD_PROTOCOL = "ALTSRP_GCM_AES128_REKEY";
   private static final int KEY_LENGTH = AltsChannelCrypter.getKeyLength();
+  // Frame size negotiation extends frame size range to [MIN_FRAME_SIZE, MAX_FRAME_SIZE].
+  private static final int MIN_FRAME_SIZE = 16 * 1024;
```

Comment:
Currently max_frame_size is defined in AltsTsiFrameProtector.java.
Please define min/max in AltsTsiFrameProtector.java.

Reply:
In AltsTsiFrameProtector.java, we could have 4 constants

MIN_FRAME_SIZE, 16KB
MAX_FRAME_SIZE, 128KB
DEFAULT_FRAME_SIZE, 16KB
LIMIT_MAX_ALLOWED_FRAME_SIZE, 1MB

We may need to change internal code to sync up.

Reply:
Do we need both MIN_FRAME_SIZE and DEFAULT_FRAME_SIZE?
I think for consistency sake, MIN_FRAME_SIZE name would be preferred. But that would involve updating MAX_ALLOWED_FRAME_BYTES to MIN_FRAME_SIZE, and wherever the getter is called. If the variable name modification seems fine, we could have just 3 constants.

Reply:
3 constants are fine. 
We need 4 getter functions to be backward compatible.

Reply:
I am using just 3 getters. 

For Loas2TsiHandshaker.java, i will cherrypick this CL, and include the update there.